mobile/app-identifier.ts

```diff
@@ -193,6 +193,32 @@ export class IOSNativeScriptCompanionAppIdentifier implements Mobile.IAppIdentif
 
 }
 
+export class WP8CompanionAppIdentifier implements Mobile.IAppIdentifier {
+	get appIdentifier(): string {
+		return "";
+	}
+
+	get deviceProjectPath(): string {
+		return "";
```

Comment:
Is this really empty string? Or maybe it is not used at all when it's inside the companion app?